bot/cogs/moderation/scheduler.py

```diff
@@ -84,12 +84,15 @@ def mod_log(self) -> ModLog:
         icon = utils.INFRACTION_ICONS[infr_type][0]
         reason = infraction["reason"]
         expiry = infraction["expires_at"]
+        expiry_at = expiry
```

Comment:
I think we can safely override `expiry` and add the duration without altering the rest of the cog.

Reply:
You just read my mind, I'm doing a refactor to get it straight from infraction['expires_at']

Reply:
Aaaaand done, now it's minimum changes to the Cog!